source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -0,0 +1,44 @@
+.. _compass-ai-data-usage:
+
+=============================
+AI and Data Usage Information
+=============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+
+Querying with natural language in Compass is powered by Generative AI
+(Gen AI), and may give inaccurate responses. See our `Generative AI FAQ 
+<https://dochub.mongodb.org/core/faq-ai-features>`__ 
+for more information about Gen AI in MongoDB products.
+
+Third Party Providers
+---------------------
+
+This feature currently uses the `Azure OpenAI Service
+<https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+hosted by Microsoft. 
+
+How Your Data is Used
+---------------------
+
+When you query with natural language in Compass, the schema of the 
+collection you are querying including the collection name, field names, 
+and types. 
```

Comment:
What we have [approved from legal ](https://docs.google.com/document/d/1iHhXescsma0lkrSUQ49niehpJP1unehc4JkzgSuK5Xw/edit)is: 

```suggestion
When you query with natural language in Compass, the following information is sent to MongoDB’s backend and/or the third party AI provider:
* The schema of the collection you are querying (including collection name, field names, and types). 
```
Since until GA this will be a single bullet I'm okay with removing the bullet but the content needs to remain basically the same.